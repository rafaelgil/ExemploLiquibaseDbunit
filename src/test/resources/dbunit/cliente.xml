<?xml version="1.0" encoding="UTF-8"?>

<dataset>

	<CIDADE CODIGO="1" NOME="MARINGA" />
	<CLIENTE CODIGO="1" NOME="RONALD A. WALLACE" CPF="495.680.041-06" RG="20.050.062-4" CIDADE="1"/>
	
</dataset>