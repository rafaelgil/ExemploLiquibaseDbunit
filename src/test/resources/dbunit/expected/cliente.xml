<?xml version="1.0" encoding="UTF-8"?>

<dataset>

	<CIDADE CODIGO="1" NOME="MARINGA" />
	<CLIENTE CODIGO="1" NOME="RONALD A. WALLACE" CPF="495.680.041-06" RG="20.050.062-4" CIDADE="1"/>
	<CLIENTE CODIGO="10" NOME="EMANUEL YURI CARLOS EDUARDO CAMPOS" CPF="691.152.720-10" RG="22.223.317-5"/>
	
</dataset>